cinema/tests/test_movie_api.py

```diff
@@ -42,21 +40,15 @@
             self.assertEqual(movies.data[0][field], titanic[field])
 
     def test_get_movies_with_genres_filtering(self):
-        movies = self.client.get(
-            f"/api/cinema/movies/?genres={self.comedy.id}"
-        )
+        movies = self.client.get(f"/api/cinema/movies/?genres={self.comedy.id}")
```

Comment:
In `test_get_movies_with_genres_filtering`, ensure that the genre IDs used in the query string are valid and exist in the database. This will help avoid false positives in the test results.